ebpf/c/tracer-ebpf.c

```diff
@@ -28,47 +28,23 @@
 /* This is a key/value store with the keys being an ipv4_tuple_t for send & recv calls
  * and the values being the struct conn_stats_ts_t *.
  */
-struct bpf_map_def SEC("maps/udp_stats_ipv4") udp_stats_ipv4 = {
+struct bpf_map_def SEC("maps/conn_stats_ipv4") conn_stats_ipv4 = {
 	.type = BPF_MAP_TYPE_HASH,
 	.key_size = sizeof(struct ipv4_tuple_t),
 	.value_size = sizeof(struct conn_stats_ts_t),
-	.max_entries = 32768,
-	.pinning = 0,
-	.namespace = "",
-};
-
-/* This is a key/value store with the keys being an ipv4_tuple_t for send & recv calls
- * and the values being the struct conn_stats_ts_t *.
- */
-struct bpf_map_def SEC("maps/udp_stats_ipv6") udp_stats_ipv6 = {
-	.type = BPF_MAP_TYPE_HASH,
-	.key_size = sizeof(struct ipv6_tuple_t),
-	.value_size = sizeof(struct conn_stats_ts_t),
-	.max_entries = 32768,
-	.pinning = 0,
-	.namespace = "",
-};
-
-/* This is a key/value store with the keys being an ipv4_tuple_t for send & recv calls
- * and the values being the struct conn_stats_ts_t *.
- */
-struct bpf_map_def SEC("maps/tcp_stats_ipv4") tcp_stats_ipv4 = {
-	.type = BPF_MAP_TYPE_HASH,
-	.key_size = sizeof(struct ipv4_tuple_t),
-	.value_size = sizeof(struct conn_stats_ts_t),
-	.max_entries = 32768,
+	.max_entries = 65536,
```

Comment:
👍 in the future we'll want to programmatically change this based on system limits

Reply:
:+1: